src/System.Composition.sln

```diff
@@ -0,0 +1,97 @@
+﻿
```

Comment:
For all the other projects, each NuGet package corresponds to a single assembly (AFAIK).  For System.Composition, right now all the assemblies are shipped in [one NuGet package](http://www.nuget.org/packages/Microsoft.Composition/).  So it doesn't really fit the existing pattern.